content.js

```diff
@@ -0,0 +1,18 @@
+chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
```

Comment:
This should be moved into the existing listener in the script.js file, before `const resultData = JSON.parse(message.action);`